internal/metrics/alerts/collector.go

```diff
@@ -0,0 +1,75 @@
+package alerts
+
+const CollectorPrometheusAlert = `
+"groups":
+- "name": "logging_collector.alerts"
+  "rules":
+  - "alert": CollectorNodeDown
+    "annotations":
+      "message": "Prometheus could not scrape {{ $labels.container }}-{{ $labels.app_kubernetes_io_name }} for more than 10m."
+      "summary": "Collector cannot be scraped"
+    "expr": |
+      up{app_kubernetes_io_component = "collector", container = "collector"} == 0 or absent(up{app_kubernetes_io_component="collector", container="collector"}) == 1
+    "for": "10m"
+    "labels":
+      "service": "collector"
+      "severity": "critical"
+      "namespace": "openshift-logging"
+  - "alert": "CollectorHighErrorRate"
```

Comment:
This alert is fine as translated now but may change based on the investigation that @vparfonov has been doing